app/models/uploaded_file.rb

```diff
@@ -1,5 +1,5 @@
 require "flex_deployment_client/middleware/uploaded_file_middleware"
-require "flex_deployment_client/requestor/uploaded_file_requestor"
```

Comment:
In order for this to be used, it would need to be set as the `self.requestor_class` on the model, so I think it's not in use